zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/publishing/PostPublisher.java

```diff
@@ -79,7 +79,6 @@ public class PostPublisher {
      * @throws IOException
      */
     public static boolean postPublish(Zebedee zebedee, Collection collection, boolean skipVerification, CollectionReader collectionReader) throws IOException {
-
```

Comment:
Small one, but I would revert this change, just so that the git diff only shows the relevant files that have been changed.